src/main/mw.c

```diff
@@ -100,7 +100,7 @@ int16_t headFreeModeHold;
 int16_t telemTemperature1;      // gyro sensor temperature
 static uint32_t disarmAt;     // Time of automatic disarm when "Don't spin the motors when armed" is enabled and auto_disarm_delay is nonzero
 
-extern uint8_t dynP8[3], dynI8[3], dynD8[3];
+extern uint8_t dynP8[3], dynI8[3], dynD8[3], PIDscaler[3];
```

Comment:
This should be moved to some header file that is included both in mw.c and pid.c. As implemented now, if one declaration changes, compiler will issue no warning, leading to hard to find error.